Assets/HandTracking/Scripts/FeatureUsageHandJointsManager.cs

```diff
@@ -56,5 +56,154 @@ private static void UpdateHandJoints(InputDeviceCharacteristics flag, Hand hand)
                 hand?.DisableHandJoints();
             }
         }
+
+        private class Hand
+        {
+            public Hand(GameObject handJointPrefab)
+            {
+                this.handJointPrefab = handJointPrefab;
+                handRoot = new GameObject("HandParent");
```

Comment:
Should we destroy these GameObjects? I think I've seen these accumulate over time, especially in some scenarios where a scene stays loaded for the duration of the app lifetime while others are loaded additively.

Reply:
Oh good call!

Reply:
Added cleanup code